Makefile

```diff
@@ -1,56 +1,67 @@
+# Default to using regular `cargo`. The `ci` target will override this.
```

Comment:
```suggestion
# Default to using regular `cargo`. CI targets may override this.
```
just to be more precise, given now we have two ci entrypoints, and both override it... but maybe this changes in the future (and we may forget to update this)